docker-compose.yml

```diff
@@ -51,6 +47,8 @@ services:
   ipfs-cluster-0:
     profiles: ["ipfs"]
     container_name: ipfs-cluster-0
+    networks:
```

Comment:
The main network doesn't have any explicit configuration here. Is there an advantage to using a specific named network rather than using a "default" network created by docker compose?

Reply:
yes, with this config we can easily add connect to the Hyperledger Fabric test-network Docker network, called fabric_test.  One line change.